elements/rh-button/demo/index.html

```diff
@@ -9,8 +9,9 @@
     <!-- uncomment the es5-adapter if you're using the umd version -->
     <script src="/@webcomponents/webcomponentsjs/custom-elements-es5-adapter.js"></script>
     <script src="/@webcomponents/webcomponentsjs/webcomponents-loader.js"></script>
-    <script src="https://cdnjs.cloudflare.com/ajax/libs/require.js/2.3.5/require.min.js"></script>
-    <script>require(['../rh-button.umd.js'])</script>
+
+    <script src='../../rhelement/rhelement.umd.js'></script>
```

Comment:
@mwcz - I assume that this is the pattern we want developers to follow?

Reply:
I don't think we should express a preference.  We're offering ESM, AMD, CJS, and globals so developers can choose the module system that's best for their project.